container_pipeline/workers/worker_start_scan.py

```diff
@@ -0,0 +1,632 @@
+        This scans the images for the job requests in start_scan tube.
+        this cals the ScannerRunner for performing the scan work
+        """
+
+        debug_logs_file = os.path.join(
+            job_info["logs_dir"], "service_debug.log")
+        dfh = config.DynamicFileHandler(logger, debug_logs_file)
+
+        logger.info('Got job: %s' % job)
+        scan_runner_obj = ScannerRunner(job)
+        status, scanners_data = scan_runner_obj.scan()
+        if not status:
+            logger.critical(
+                "Failed to run scanners on image under test, moving on!"
+            )
+        else:
+            logger.info(str(scanners_data))
+
+        # Remove the msg and logs from the job_info as they are not
+        # needed now
+        scanners_data.pop("msg", None)
+        scanners_data.pop("logs", None)
+        scanners_data.pop("scan_results", None)
+
+        # if weekly scan, push the job for notification
+        if job.get("weekly"):
+            scanners_data["action"] = "notify_user"
+            self.queue.put(json.dumps(scanners_data), 'master_tube')
+        else:
+            # now scanning is complete, relay job for delivery
+            # all other details about job stays same
+            # change the action
+            scanners_data["action"] = "start_delivery"
+            # Put the job details on central tube
+            self.queue.put(json.dumps(scanners_data), 'master_tube')
+            logger.info(
+                "Put job for delivery on master tube"
+            )
+
+        # remove per file build log handler from logger
+        if 'dfh' in locals():
+            dfh.remove()
+
+
+    #
+    # def handle_build_success(self, job):
+    #     """Handle build success for job"""
+    #     self.logger.debug("Build is successful going for next job")
+    #
+    # def handle_build_failure(self, job):
+    #     """Handle build failure for job"""
+    #     self.queue.put(json.dumps(job))
+    #     self.logger.info(
+    #         "Build is not successful putting it to failed build tube")
+    #     appid = job['appid']
+    #     jobid = job['jobid']
+    #     desired_tag = job['desired_tag']
+    #     project = appid + "/" + jobid + ":" + desired_tag
+    #     build_logs_file = os.path.join(job['logs_dir'], 'build_logs.txt')
+    #     self.notify_build_failure(
+    #         get_job_name(job), job['notify_email'], build_logs_file,
+    #         project, job['jobid'], job['TEST_TAG'])
+
+
+if __name__ == '__main__':
+    DOCKER_HOST = "127.0.0.1"
+    DOCKER_PORT = "4243"
+
+    load_logger()
+    logger = logging.getLogger("scan-worker")
+
+    SCANNERS_OUTPUT = {
+        "registry.centos.org/pipeline-images/pipeline-scanner": [
+            "image_scan_results.json"],
+        "registry.centos.org/pipeline-images/scanner-rpm-verify": [
+            "RPMVerify.json"]}
+
+    try:
+        # docker client connection to CentOS 7 system
+        conn = docker.Client(base_url="tcp://%s:%s" % (
```

Comment:
it'd be good not to use global vars. The ``conn`` object could be passed to ``ScanWorker`` instance and can be used from there, as needed.